src/data/showcases.js

```diff
@@ -1104,6 +1104,15 @@ export const Showcases = [
     source: "https://github.com/cardano-foundation/cardano-governance-voting-tool",
     tags: ["governance", "opensource"],
   },
+  {
+    title: "TokeoPay",
+    description:
+      "Your go-to multichain smart wallet on Cardano.",
```

Comment:
```suggestion
      "Your go-to multichain smart wallet on Cardano.",
```
Sometimes marketing language can be clipped into these descriptions but "go-to" indicates a bias which we cannot have on the Developer Portal.  I would suggest replacing that phrase with a list of distinctive features: especially those favouring users with a common interest in Cardano and Bitcoin.